### DISCUSSION TITLE
133.0 ETP Question

### DISCUSSION THREAD

**Discussion by brzzdev** [yokoffing/Betterfox#351] [General]
Just saw in the release notes for 133.0
`There are issues related to Strict ETP recently. Switching to Standard tracking protection should solve most issues.`

Would it be possible to get further context here? I don't think I've seen anything, but I'd love to be able to track any issues, so I know when it's advisable to switch back to strict.

**Comment by yokoffing** [comment]
I haven't bookmarked specific complaints this time. This is based on general user feedback I've come across over the past weeks from users of Strict ETP (not always Betterfox specifically).

For awhile now, Mozilla did a great job fixing Strict compatibility with many sites, but recent builds seem to have regressed.

**Reply by brzzdev** [reply]
I see, thank you
